### DISCUSSION TITLE
Cannot generate request documentation

### DISCUSSION THREAD

**Discussion by baharrast** [dedoc/scramble#491] [Q&A]
These are my route list:
![Screenshot 2024-08-19 122634](https://github.com/user-attachments/assets/53f302d8-ad83-4189-9a0b-38555b4b7fd7)
and this is my update function in ClubPointSettingController.php
![Screenshot 2024-08-19 122841](https://github.com/user-attachments/assets/f1a94bb7-99fb-469d-949b-9555c8df197b)
now i get error that 
![Screenshot 2024-08-19 135628](https://github.com/user-attachments/assets/9c914f4f-c767-41af-a6b3-76e6a9a06107)
whereas Atrialab\Club\Models\ClubPointSetting is my model and my request is Atrialab\Club\Http\Requests\Admin\ClubPointSetting\V1\UpdateRequest
The scramble call model instead of form request
how i can fix this error?

**Comment by romalytvynenko** [comment]
@baharrast which version of Scramble are you on?

**Reply by baharrast** [reply]
v0.11.11

**Comment by romalytvynenko** [comment]
@baharrast You can try switching the order of arguments, but this still is a bug and I will fix it

**Comment by baharrast** [comment]
Thanks
